composer.json

```diff
@@ -49,5 +49,11 @@
         "branch-alias": {
             "dev-master": "1.x-dev"
         }
+    },
+    "scripts": {
+        "check": [
+            "@test"
+        ],
+        "test": "phpunit --colors=always"
```

Comment:
I don't normally rely on these. But don't have a problem. Can you add a doc portion so people know how to use it in README.